tee-worker/core/parentchain/light-client/src/light_validation.rs

```diff
@@ -48,29 +48,25 @@ impl<Block: ParentchainBlockTrait, OcallApi: EnclaveOnChainOCallApi>
 	pub fn new(
 		ocall_api: Arc<OcallApi>,
 		finality: Arc<Box<dyn Finality<Block> + Sync + Send + 'static>>,
+		light_validation_state: LightValidationState<Block>,
 	) -> Self {
-		Self {
-			light_validation_state: LightValidationState::new(),
-			ocall_api,
-			finality,
-			ignore_validation_until: 0u32.into(),
-		}
+		Self { light_validation_state, ocall_api, finality, ignore_validation_until: 0u32.into() }
 	}
 
-	fn initialize_relay(
-		&mut self,
-		block_header: Block::Header,
-		validator_set: AuthorityList,
-	) -> Result<RelayId, Error> {
-		let relay_info = RelayState::new(block_header, validator_set);
+	// fn initialize_relay(
+	// 	&mut self,
+	// 	block_header: Block::Header,
+	// 	validator_set: AuthorityList,
+	// ) -> Result<RelayId, Error> {
+	// 	let relay_info = RelayState::new(block_header, validator_set);
```

Comment:
Why comment out instead of removal? Does upstream do so?

Reply:
removed